src/media/js/addon/reducers/dashboard.js

```diff
@@ -15,6 +16,31 @@ const initialState = {
 
 export default function addonDashboardReducer(state=initialState, action) {
   switch (action.type) {
+    case addonActions.FETCH_OK: {
```

Comment:
If it's not too big a burden, I would really love to document our reducers' expected payload signatures.